game/factions/faction_loader.py

```diff
@@ -0,0 +1,30 @@
+import json
+import os
+import logging
+
+from game.factions.faction import Faction
+
+FACTION_DIRECTORY = "./resources/factions/"
+
+
+def load_factions() -> {str, Faction}:
+    files = os.listdir(FACTION_DIRECTORY)
```

Comment:
Probably best to use `pathlib.Path` for new code. More robust with mypy and a lot more usable imo.